azure/ConsiderationsForServiceDesign.md

```diff
@@ -141,94 +144,206 @@ cannot collide with a resource path that contains user-specified resource ids.
+A status monitor is created to track the additional processing and the ID of the status monitor
+is returned in the `Operation-Id` header of the response.
+The response may also include an `Operation-Location` header for backward compatibility.
+If the resource supports ETags, the response may contain an `etag` header and possibly an `etag` property in the resource.
+
+```text
+HTTP/1.1 201 Created 
+Operation-Id: 22
+Operation-Location: https://items/operations/22
+etag: "123abc"
+
+{
+  "id": "FooBar", 
+  "etag": "123abc",
+  "prop1": 555,
+  "prop2": "something"
+}
+```
 
-5. After the operation processing completes, a GET request to status monitor URL returns a response with a status field containing a terminal value -- `Succeeded`, `Failed`, or `Canceled` -- that indicates the result of the operation.
-If the status is `Failed`, the status monitor resource must contain an `error` field with a `code` and `message` that describes the failure.
-If the status is `Succeeded`, the response may contain additional fields as appropriate, such as results
-of the operation processing.
+The client will issue a GET to the status monitor to obtain the status of the operation performing the additional processing.
 
-An important distinction between RELO and status monitor LROs is that there is a unique status monitor for each
-status monitor LRO, whereas the status of all RELO operations is combined into the status of the resource.
-So status monitor LROs are "one-to-one" with their operation status, whereas RELO-style LROs are "many-to-one".
+```text
+GET https://items/operations/22?api-version=2022-05-01
+```
+
+When the additional processing completes, the status monitor will indicate if it succeeded or failed.
+
+```text
+HTTP/1.1 200 OK
+ 
+{
+   "id": "22",
+   "status": "Succeeded"
+}
+```
+
+If the additional processing failed, the service may delete the original resource if it is not usable in this state.
+
+### Long-running delete operation
+
+A long-running delete operation follows the general pattern of a long-running operation --
+it returns a `202 Accepted` with a status monitor which the client uses to determine the outcome of the delete.
+
+The resource being deleted should remain visible (returned from a GET) until the delete operation completes successfully.
+
+When the delete operation completes successfully, a client must be able to create new resource with same name without conflicts.
+
+### Controlling a long-running operation
+
+It might be necessary to support some control action on a long-running operation, such as cancel.
+This is implemented as a POST on the status monitor endpoint with `:action` added.
+
+```text
+POST /<status-monitor-url>:cancel?api-version=2022-05-01
+```
+
+A successful response to a control operation should be a `200 OK` with a representation of the status monitor.
+
+```text
+HTTP/1.1 200 OK 
+ 
+{
+   "id": "22",
+   "status": "Canceled"
+}
+```
```

Comment:
Do we want prescriptive guidance on the path structure for listing all active operations?  Either globally or per operation?  Or is that coming in a future update?